fake-api/templates/imposters.ejs

```diff
@@ -0,0 +1,6 @@
+{
+    "imposters": [
+        <% include node-contract.json %>,
```

Comment:
I'm assuming place holders?

Reply:
ejs, those actually import the JSON directly inline. We load this file as the main configuration file in the docker image, and it pulls in the node/pod-contract files.